From 7a18844223d40b684e2f24a02741a1bd53048218 Mon Sep 17 00:00:00 2001
Message-Id: <7a18844223d40b684e2f24a02741a1bd53048218.1650802871.git.geliang.tang@suse.com>
From: Evgeniy Litvinenko <evgeniyl@fb.com>
Date: Fri, 23 Jul 2021 15:36:45 -0700
Subject: [PATCH] selftests/bpf: Document vmtest.sh dependencies
Git-commit: 7a18844223d40b684e2f24a02741a1bd53048218
Patch-mainline: v5.15-rc1
References: bsn#127
Modified-by-SEL: No


Add a list of vmtest script dependencies to make it easier for new
contributors to get going.

Signed-off-by: Evgeniy Litvinenko <evgeniyl@fb.com>
Signed-off-by: Andrii Nakryiko <andrii@kernel.org>
Link: https://lore.kernel.org/bpf/20210723223645.907802-1-evgeniyl@fb.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/bpf/README.rst | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/tools/testing/selftests/bpf/README.rst b/tools/testing/selftests/bpf/README.rst
index 8deec1ca9150..9b17f2867488 100644
--- a/tools/testing/selftests/bpf/README.rst
+++ b/tools/testing/selftests/bpf/README.rst
@@ -19,6 +19,13 @@ the CI. It builds the kernel (without overwriting your existing Kconfig), recomp
 bpf selftests, runs them (by default ``tools/testing/selftests/bpf/test_progs``) and
 saves the resulting output (by default in ``~/.bpf_selftests``).
 
+Script dependencies:
+- clang (preferably built from sources, https://github.com/llvm/llvm-project);
+- pahole (preferably built from sources, https://git.kernel.org/pub/scm/devel/pahole/pahole.git/);
+- qemu;
+- docutils (for ``rst2man``);
+- libcap-devel.
+
 For more information on about using the script, run:
 
 .. code-block:: console
-- 
2.26.2

